packages/gafl-webapp-service/src/pages/contact/address/select/address-select.njk

```diff
@@ -32,6 +32,7 @@
 
     {{ govukRadios({
         idPrefix: "address",
+        classes: "capitalise",
```

Comment:
Semantic class names should be preferred, so use 'address_list' rather than 'capitalise'